### ISSUE TITLE
Is it possible to link a database to a library?

### ISSUE THREAD

**Issue by nvd05** [TalAloni/SMBLibrary#203]
Hello TalAloni.
Is it possible to link a library to a database?

In other words:

1) Possibility to modify the authorization logic so that this can be linked to the database.

2) Possibility to link the logging system to the database (this is already implemented).

3) Ability to manage public folders through the database:
1. Change the list of public folders when the server is running.
2. Change folder settings when the server is running.

In other words - give the ability to change parts of your code (without changing the code of the library itself).
For example, through function overrides.
Standardly it is one logic, but you can change it to suit your needs.

From the point of view of queries, of course, it is not very little, but not very much either.

**Comment by TalAloni** [comment]
Yes, certainly possible and have been done by users of this library.
